exporters/tests/test_datetime.py

```diff
@@ -102,3 +105,29 @@ def test_github():
     actual_unix = github.parse_datetime(TIMESTRING).timestamp()
 
     assert actual_unix == EXPECTED_UNIX
+
+
+@pytest.mark.parametrize(
+    "timestamps, expected",
+    [
+        ("1652305808.000", "1652305808.0"),
+        ("1652305808.122", "1652305808.0"),
+        ("1652305808", "1652305808.0"),
+        ("1652305808.0", "1652305808.0"),
+    ],
+)
+def test_to_epoch_from_string(timestamps, expected):
+    epoch_from_str = to_epoch_from_string(timestamps)
+    assert str(epoch_from_str.timestamp()) == expected
+
+
+@pytest.mark.xfail(raises=ValueError)
```

Comment:
`xfail` is meant for tests that are _temporarily_ failing, because you want to acknowledge the existence of a bug, or because something isn't implemented yet.

To test that an exception is raised, you'd do

```python
with pytest.raises(ValueError):
    to_epoch_from_string(timestamps)
```

Though  it seems like a nit, it _does_ affect test output.